slack.el

```diff
@@ -120,7 +120,7 @@ never means never show typing indicator."
         (mapcar #'(lambda (data)
                     (slack-room-create data team class))
                 (append
-                 (cl-remove-if-not #'(lambda (data) (plist-get data :is_open)) datum)
+                 (cl-remove-if-not #'(lambda (data) (eq (plist-get data :is_open) :json-true)) datum)
```

Comment:
In my `json.el` (version 1.4) returns `t` not `json-true` when true in JSON.
Is there any difference between versions?